microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
-    at async b (devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js:1:4438)
-    at async #v (devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js:1:8182)
-
-Source modules:
-    devtools://devtools/bundled/panels/elements/elements.js 22994ef71c62ba25b9494e26da78b621c5995bff1830c183035b4c1eb7c6b12b
-    devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js bac7011989ae5c7870c7f2964e88a51306eb2d520236329f09be5f5850105a42
-```
+   ```bash
+   npm install @microsoft/edge-devtools-crash-analyzer-support
+   ```
 
-The additional metadata is the line `Source modules:` and the lines below it.  Each line in that section is parsed as follows:
+1. Import the library in your JavaScript code, and then call the `installErrorStackModuleAnnotations` function:
 
-* The line must end with a 64-character line of hexadecimal characters corresponding to the SHA-256 hash of the script being executed.  This is a byte-for-byte hash of the content for each JavaScript file.
+   ```javascript
+   import { installErrorStackModuleAnnotations } from '@microsoft/edge-devtools-crash-analyzer-support';
 
-* The rest of the line is trimmed and treated as the module name, such as `/panels/elements/elements.js`.  The module name should correspond to file names listed in the main part of the stack trace.
+   installErrorStackModuleAnnotations(Error);
+   ```
 
 
 <!-- ====================================================================== -->
-## Use Application Insights to receive a stack trace from the web browser
+## Analyze stack traces in the Crash analyzer tool
 
-To use the **Crash analyzer** tool, you first use a production stack trace collection system, to receive the stack trace from a web browser that's running your app.  One such production stack trace collection system is the Application Insights feature of Azure Monitor.  Application Insights performs Application Performance Management (APM) for live web applications.
+The **Crash analyzer** tool in DevTools unminifies the specially-formatted stack traces that you collected from users in production. If your source maps also contain the original source code, the **Crash analyzer** tool will show you the original file names and function names that make up the stack trace.
 
-See [Application Insights overview](/azure/azure-monitor/app/app-insights-overview).<!-- https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview -->
 
+<!-- ------------------------------ -->
+#### Make source maps accessible in DevTools
 
-<!-- ====================================================================== -->
-## Analyze crash stack traces in the Crash analyzer tool
+The **Crash analyzer** tool uses the **Source module** information in the stack trace to retrieve the source maps that correspond to the script files in the stack trace, this means that you need to make your source maps accessible in DevTools.
```

Comment:
comma before "this" is ungramm., will revise.

Reply:
* added first "that's"
* broke into 2 sentences.

The **Crash analyzer** tool uses the **Source module** information that's in the stack trace to retrieve the source maps that correspond to the script files in the stack trace.  This means that you need to make your source maps accessible in DevTools.